src/styles/common.css

```diff
@@ -330,7 +217,47 @@ dt {
   gap: var(--d);
 }
 
-/* test-card  */
+.skip {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+
+.skip:focus {
+  position: static;
+  width: auto;
+  height: auto;
+}
+
+.centre {
+  text-align: center;
+}
+
+.motto {
+  font-size: 1.2em;
+}
+
+.copyright {
+  font-size: calc(var(--footer-height) / 9);
+  text-align: right;
+}
+
+.header-text {
+  font-size: 1em;
+}
+
+@media (width <= 400px) {

```

Comment:
Or e.g. 

```
.header-logo {
  height: var(--logo-height);
  display: none;
}

@media (width > 480px) {
  .header-logo {
    display: block;
  }
}
```

Reply:
Alright, mobile first makes sense. Will correct this.